wheels/model/errors.cfm

```diff
@@ -37,6 +37,47 @@ public array function allErrors() {
 	return variables.wheels.errors;
 }
 
+/**
+ * Gets all associated errors recursively
+ *
+ * [section: Model Object]
+ * [category: Error Functions]
+ */
+public array function allAssociationErrors(array seen = [], boolean associated = false, string returnAs = "") {
```

Comment:
Is this name is ideal @neokoenig @dbelanger

Reply:
I guess the question is should it be part of the public API? presumably a developer will only be interested in the parent model - ie using `parentModel.allErrors(), parentModel.valid()` or outputting errors as part of a nested properties form, i.e, `errorMessagesFor(user.address)` - I'm not necessarily against it, just thinking out loud.

Reply:
@neokoenig I was thinking of adding an argument to allErrors to allow for returning associated errors as well, then could privatise allAssociationErrors

Reply:
I have modified it so allErrors has an includeAssociations argument and allAssociationErrors is private

Reply:
The `includeAssociations` is a good solution.. I can't find an obvious equivalent in Rails

Reply:
Obviously at the moment all the errors get flattened and so you can't tell which error belongs to which association or even which model, which is why I originally included a returnAs == "structs" thing so it was `{ modelName: variables.wheels.class.modelName, errors: variables.wheels.errors }` so you could differentiate the errors